paper-icon-button.html

```diff
@@ -76,7 +76,11 @@
         position: relative;
         padding: 8px;
         outline: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+        /* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
```

Comment:
Given that we have precedent for including both, I'll go with it. Maybe later we'll discover we can remove one.